### DISCUSSION TITLE
Allow Anonymous Mode to load >100 subscriptions

### DISCUSSION THREAD

**Discussion by MetalNeo** [Docile-Alligator/Infinity-For-Reddit#1247] [Ideas]
Anonymous Mode lets you subscribe to as many subreddits as you want, but will not load the frontpage if you have too many subscriptions. It would be nice if the app were able to either load the subscriptions in chunks or pick 100 randomly from the list and load just those.

**Comment by Docile-Alligator** [comment]
That's a nice suggestion!
